client/context/context.go

```diff
@@ -295,13 +295,13 @@ func GetFromFields(input io.Reader, from string, genOnly bool) (sdk.AccAddress,
 		return addr, "", nil
 	}
 
-	keybase, err := keys.NewKeyring(sdk.KeyringServiceName(),
+	keybase, err := keyring.NewKeyring(sdk.KeyringServiceName(),
```

Comment:
FTR my future plan entails renaming this function to `New()`

Reply:
Makes sense